docs/Course Wiki/CS Course Offerings/CS421.md

```diff
@@ -0,0 +1,56 @@
+# CS421
+
+CS421 (Programming Languages and Compilers) is a 3-credit hour course that counts as an Advanced Computing elective and is explicitly required for CS majors.
+## Content Covered
+
+This course teaches students functional programming, and the theory and implementation behind interpretes / compilers. Student's learn OCaml, and use it for all assignments and midterms. 
```

Comment:
"interpreters" is misspelled.  Also can you move this paragraph below all the bullet points?